exercises/resistor-color-trio/description.md

```diff
@@ -0,0 +1,43 @@
+If you want to build something using a Raspberry Pi, you'll probably use _resistors_. For this exercise, you need to know only three things about them:
+
+- Each resistor has a resistance value.
+- Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
+  To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values.
+- Each band acts as a digit of a number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
+  In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take 3 colors as input, and outputs the correct value, in Ohms.
+  The color bands are encoded as follows:
+
+* Black: 0
+* Brown: 1
+* Red: 2
+* Orange: 3
+* Yellow: 4
+* Green: 5
+* Blue: 6
+* Violet: 7
+* Grey: 8
+* White: 9
+
+In `resistor-color duo` you decoded the first two colors. For instance: orange-orange got the main value `33`.
+The third color stands for how many zeros need to be added to the main value. The main value plus the zeros gives us a value in Ohms.
+For the exercise it doesn't matter what Ohms really are.
+For example:
+
+- orange-orange-black would be 33 and no zeros, which becomes 33 Ohms.
+- orange-orange-red would be 33 and 2 zeros, which becomes 3300 Ohms.
+- orange-orange-orange would be 33 and 3 zeros, which becomes 33000 Ohms.
+
+(If Math is your thing, you may want to think of the zeros as exponents of 10. If Math is not your thing, go with the zeros. It really is the same thing, just in plain English instead of Math lingo.)
+
+This exercise is about translating the colors into a sentence:
+
+> "The value of this resistor is … Ohms."
+
+So an input of `“orange”, “orange”, “black”` should return
+
+> “The value of this resistor is 33 Ohms.”
+
+The trick is that when we talk about Ohms, as soon as we get more than a thousand, we say “kilo”, in the same way that we say “kilograms” (kg) and “kilometers” (km). In the same way, we get “kilo-ohms” (kOhms).
```

Comment:
For consistency and readability I'd prefer either writing out the unit completely (`kilo Ohms`) or in short form (`kΩ`), but not mixed.

I think you're using the unicode quotation marks here `“”` instead of `""`, is that intended?

Reply:
Thanks Sascha! 
> writing out the unit completely (`kilo Ohms`) or in short form (`kΩ`), but not mixed.

We want to avoid special characters in this one, so let's go with 'kilo Ohms'.

>Using kilograms as example is slightly misleading 
I understand your reasoning, but since it's just about explaining where the 'kilo'-part is coming from in a non-mathy way, and since the base unit is not really relevant for this particular exercise, I'd rather leave it as is. Can you live with that?

> Quotation marks: not intended.

Reply:
> Can you live with that?

Yes, of course. All of my comments were meant as suggestions that you may or may not consider, not requests for change. I should've made that clearer perhaps. :)